### PR TITLE
Removes latejoin viruses.

### PR DESCRIPTION
**Pull request by Irradiation**
Due to (probably?) a vocal minority complaining about it, I'm making this PR to see if people actually want this mechanic gone.

t. medbay main who doesn't care (or play often) either way.

Hopefully there will be a discussion and not just emojicracy, but who am I kidding.

===============================================================
**UPDATE**
Right well, I am apparently a dumbass who can't read properly.

Roundstart viruses aren't really a thing and what people call "roundstart" viruses is just latejoiners and mice pozzing the fuck out of everyone.

That said, do you guys still want Latejoin viruses to exist?
-If you do, do you want to try the "soft" approach and fiddle with syndrome danger levels/weight to make some shit (like Glasgow, Bees, etc) not happen or happen very rarely?
===============================================================
[discussion][gameplay]

:cl:
 * rscdel: Removes latejoin viruses.

### CODE REVIEW

**Comment by adu6oo** [comment]
didn't think i'd get this far
anyways i feel like roundstart viruses really detract so much from the rounds, especially in latepop
while yes the viruses typically arent THAT strong sometimes it can be really unbearable, had a round where it was roundstart horsehead and that was a total shitshow, then it happens in latepop with no virology and a skeleton medbay and it ends up 10 minutes into the round everybody keeps throwing up bees
it's just entirely obnoxious to have this type of thing seemingly make so many rounds instantly very dumb because everybody gets infected by some dumb virus which yeah theyre easy to cure but i don't find it very exciting to take spaceacilian and then lay down for a minute every 2 rounds just so i dont get owned by bees again (again) (again) (again)

**Comment by Irradiation** [comment]
To be fair, there's also a soft way to "fix" roundstart viruses, and it's changing around the danger_level of many syndromes, dicking around with danger_level weight in the roundstart configs.

Much like Dynamic, a lot of bad events in Nu-Virus happen due to the shittiest RNG possible and I believe the shitfests are isolated elements. Probably.

**Comment by PrimeDSS13** [comment]
I'm okay with latejoiners being able to have viruses, since it makes sense. Maybe they brought it in from somewhere else, and as a gameplay perspective it discourages latejoining over readying up for round start. It also still gives Virologists a little bit of a job to do. I'm all for removing roundstart viruses from people who are at the station from 12:05 though.

**Comment by Gazathor** [comment]
the real solution is to give everyone a immunity booster before they arrive on station so they wont catch a cold from sitting on the equivalent of the public bus, waiting to get to station

**Comment by adu6oo** [comment]
yeah ive seen people say roundstart viruses should just be limited to shit like sneezing, nothing horribly game altering like throwing up bees, horsehead, and glasgow
i'd also be fine with that
also what primed said, letting latejoiners still have viruses is also fine

**Comment by Kammerjunk** [comment]
I just want to play SS13 instead of Space Uganda
well, I guess it'd be Space China now

**Comment by Gazathor** [comment]
> I just want to play SS13 instead of Space Uganda
> well, I guess it'd be Space China now

are we getting bat soup then?

**Comment by Buchse** [comment]
I honestly believe that viruses in general should be mostly things you can ignore. Things like sniffles, coughs, sore throats, coughing that doesn't turn you into a biological plague hazard for not wearing full biohazard gear, etc. If roundstart viruses aren't removed entirely, they should only have like 1-2 symptoms max, with those symptoms being almost inconsequential flavor stuff, like the aforementioned sore throats and such. Making the wacky shit like Glasgow, gyro, respiratory putrifaction, etc, a lot higher in threat might alleviate some of this.

Honestly a minor and major version of most "normal" symptoms could be another way to go about it. For example, "minor" sneezing doesn't leave puddles of biohazard all over the place, but a virus with high strength with the sneeze symptom leaves infectious mucous.

Mostly shotgunning ideas, but nobody really likes having to be a part of the viro shuffle 100% of the time, every round.

**Comment by AI-Is-Rogue** [comment]
I enjoyed them but they were honestly a bit much at times. I agree with @PrimeDSS13 that latejoiners should be able to spawn them. Maybe past the ten-minute mark?

**Comment by Irradiation** [comment]
I'm a codelet, so I'm not 100% sure if it's new_player differentiates between regular roundstart people or latejoiners.

I'll see about writing up a softer way, which is just changing danger_levels and weights around.

**Comment by Irradiation** [comment]
**UPDATE**
Right well, I am apparently a dumbass who can't read properly.

Roundstart viruses aren't really a thing and what people call "roundstart" viruses is just latejoiners and mice pozzing the fuck out of everyone.

That said, do you guys still want Latejoin viruses to exist?
-If you do, do you want to try the "soft" approach and fiddle with syndrome danger levels/weight to make some shit (like Glasgow, Bees, etc) not happen or happen very rarely?

**Comment by adu6oo** [comment]
well in this case i do think the soft approach would not be bad, just making it so nothing horrible and awful happens from latejoiners
very supportive on flat out removing memes though or making them impossibly for latejoiners to have

**Comment by SonixApache** [comment]
Roundstart virusses aren't so bad, just glasgow and bees that are annoying.
I wish they'd have more variety; longer to incubate, more or less stages, etcetera. 

Coming onto the station with the sniffles without it turning into minor annoyance turboaids wouldn't be so bad.

**Comment by Dacendeth** [comment]
The only viruses I want are ones made by players, ones from catbeasts, and ones from biohazard events.

**Comment by PrimeDSS13** [comment]
maybe make latejoiners just have a lower chance of having a virus, remove memes from mice and latejoiners, and possibly just make mice not have roundstart viruses at all.

**Comment by Irradiation** [comment]
The current latejoin virus% is 5%, though I want to say it feels higher. RNG is a cunt.

**Comment by DeityLink** [comment]
Just remove the roundstart ones for now. Leave the latejoin ones, but please by all means remove memes. Glasgow is a danger 3 symptom so it shouldn't be able to spawn at roundstart.

**Comment by DeityLink** [comment]
@adu6oo for the last fucking time horsehead can NOT be a roundstart symptom

**Comment by DamianX** [comment]
Comments are not for keeping old code around, git is. Remove the code instead

**Comment by DeityLink** [comment]
Yeah don't comment out the whole proc, just the call.

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -319,8 +319,9 @@
 		if(prob(10)) // 10% of those have a good mut.
 			H.dna.GiveRandomSE(notflags = GENE_UNNATURAL,genetype = GENETYPE_GOOD)
 
-/mob/new_player/proc/DiseaseCarrierCheck(var/mob/living/carbon/human/H)
+/mob/new_player/proc/DiseaseCarrierCheck(var/mob/living/carbon/human/H)  
 	// 5% of players are joining the station with some minor disease
+	/*  Uncomment this if somebody wants roundstart viruses again in the future.
```
Comment (Kurfursten) [inline code/modules/mob/new_player/new_player.dm]: You do realize that with this comment you are going to be perpetuating the same mistake that you suffered from

**Comment by ShiftyRail** [comment]
`code/modules/mob/new_player/new_player.dm:419:error: DiseaseCarrierCheck: undefined proc`

**Comment by ShiftyRail** [comment]
Better late than never, I guess. Yell at me if this wasn't a good choice

**Pull request merged**